### PR TITLE
add EntandoMessageCodesResolver

### PR DESCRIPTION
**Pull request by stefanopuddu**

### CODE REVIEW

**Comment by entando-service** [comment]
All is well

File: engine/src/main/java/org/entando/entando/web/common/EntandoMessageCodesResolver.java [hunk 1]
```diff
@@ -0,0 +1,64 @@
...
+import java.util.LinkedHashSet;
+import java.util.List;
+import java.util.Map;
+import java.util.Set;
+
+import org.springframework.util.StringUtils;
+import org.springframework.validation.DefaultMessageCodesResolver;
+import org.springframework.validation.MessageCodeFormatter;
+
+public class EntandoMessageCodesResolver extends DefaultMessageCodesResolver {
```
Comment (joewhite101) [inline engine/src/main/java/org/entando/entando/web/common/EntandoMessageCodesResolver.java]: Please add a high level description of this in a comment. This is pretty invisible functionality so we need to make sure to describe what's going on here.

File: engine/src/main/java/org/entando/entando/web/label/LabelController.java [hunk 2]
```diff
@@ -58,6 +58,7 @@ public void setLabelValidator(LabelValidator labelValidator) {
     @RestAccessControl(permission = Permission.SUPERUSER)
     @RequestMapping(method = RequestMethod.GET, produces = MediaType.APPLICATION_JSON_VALUE)
     public ResponseEntity<?> getLables(RestListRequest requestList) {
+        logger.debug("loading labels");
```
Comment (joewhite101) [inline engine/src/main/java/org/entando/entando/web/label/LabelController.java]: Does this requestList contain anything?

Comment (stefanopuddu) [reply engine/src/main/java/org/entando/entando/web/label/LabelController.java]: Yes, labels can be filtered by key and by value

**Comment by entando-service** [comment]
All is well

**Pull request merged**